### PR TITLE
Rename to yapenv

### PR DESCRIPTION
**Pull request by phuber92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,85 @@
...
+### Environment Variables
+
+- `VERSION_PATH`: File path to the `.version` file.
+- `SETUP_VERSION`: Version string used by `setuptools.setup.version`.
+- `YAPENV_ENV_FILE`: Env file to load when running commands (default=`.env`).
+- `YAPENV_FULL_ERRORS`: Boolean that tells `yapenv` to dump full traceback (default=`"false"`).
+- `YAPENV_CONFIG_FILES`: Array of yapenv config file names (default=`".yapenv.yaml .yapenv.yml .yapenv .yapenv.json"`).
+- `NO_COLOR`: Boolean that disables colorized logging output (default="`false`")
+- `VIRTUAL_ENV`: File path of python virtualenv (default=`None`)
+- `SHELL`: Default command line shell.
```
Comment (LamaAni) [inline README.md]: OS Level, no need to define this.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,85 @@
...
+```
+
+#### Package List Method
+```
+python_version: "3.10"
+venv_directory: .venv
+environments:
+  dev:
+    requirements:
+    - package: flake8
```
Comment (LamaAni) [inline README.md:103]: This example should include also version types. Can be 
```yaml
- package: name
- package: name=1.1.0
- name
- name=1.1.0
```

File: README.md [hunk 3]
```diff
@@ -1,39 +1,85 @@
-# yape
-Yet another python environment manager (One with less opinions)
+# yapenv
+Yet another (less opinionated) python environment
```
Comment (LamaAni) [inline README.md]:
```suggestion
### Yet Another Python Environment manager
###### * with less opinions
```

File: README.md (hunk 1 above)
Comment (phuber92) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (phuber92) [reply README.md:103]: Updated

File: README.md [hunk 4]
```diff
@@ -1,39 +1,82 @@
...
+```
+python_version: "3.10"
+venv_directory: .venv
+environments:
+  dev:
+    requirements:
+    - package: flake8
+    - package: black
+requirements:
+- package:
```
Comment (LamaAni) [inline README.md]: That is not correct

Comment (phuber92) [reply README.md]: Fixed

**Review by LamaAni — APPROVED** [review]

**Pull request merged**